src/components/SymptomTable.vue

```diff
@@ -136,11 +136,42 @@ export default {
       // this.fillData();
       console.log("storageThumbnailSets change", this.storageThumbnailSets);
     },
+    storageSymptomsTemplates: function () {
+      console.log(
+        "storageSymptomsTemplates change",
+        this.storageSymptomsTemplates
+      );
+    },
   },
   computed: {
     storageThumbnailSets() {
       return this.$store.state.thumbnailList;
     },
+    storageSymptomsTemplates() {
+      if (this.$store.state.SymptomsTemplates[0] != null) {
+        let swp = this.$store.state.selectTemplate;
+        var filterDisplayTemplate = this.$store.state.SymptomsTemplates.filter(
+          function (item) {
```

Comment:
How about refactor this part with `ES6 arrow function`, `OR operator` and `ES6 spread syntax`?

```
let filterDisplayTemplate = this.$store.state.SymptomsTemplates.filter(
    item => ({ ...item, title: swp || "commonCold" })
);
```

This will return item with `title: swp` if swp is not falsy, `title: 'commonCold` if swp is falsy

Reply:
![image](https://user-images.githubusercontent.com/16307922/92087830-98d82680-edfe-11ea-8677-837e96d86470.png)
我不是很確定....

Reply:
The solution I provided is for returning item (map), not for filter 😢 
The current solution looks good